ui/public-key.json

```diff
@@ -0,0 +1,6 @@
+{
+	"ui:widget": "textarea",
```

Comment:
This doesn't have to be a separate schema. The JSON schema spec permits using unknown properties in a schema spec which will be ignored by any validation/codegen tools running against the schema.

We should just override this inline on the data spec. I'd also suggest using `component` and `properties` (or `props` depending on how react-specific you want to get) rather than `widget` and `options` since it's more semantically accurate.
```
{
	"$id": "http://platform.selfkey.org/schema/attribute/public-key.json",
	"$schema": "http://platform.selfkey.org/schema/identity-attribute.json",
	"idAttribute": true,
	"idAttributeRepository": "http://platform.selfkey.org/repository.json",
	"title": "Public Key",
	"description": "A cryptographic public key.",
	"type": "string",
	"ui:component": "textarea",
	"ui:properties": {
		"rows": 5
	}
}
```

Reply:
You are right, it does not have to. But I prefer it that way for the following reasons:

-  makes attribute json schema cleaner and closer to standard spec
- separation of concerns
- integration in react-jsonschema-form
- allowing to reuse the same attribute with different ui schemas

Reply:
wontfix